### ISSUE TITLE
Customization of tabs bar of a nested Tabs Component causing trouble on iOS

### ISSUE THREAD

**Issue by kutoman** [codenameone/CodenameOne#3023]
In my application there are several places where I use nested `Tabs` (in turn as child of `Tabs`). In some of those I customized the tabs bar in order to hide some tab buttons which point to tab pages which must not be selectable by button click. This was working whithout any problems for a long time. But somehow (I don't know when this started) the tabs bar isn't rendered properly on iOS anymore.  The layout of the tabs bar is completely messed up only on iOS. 

You can produce this by the following code:

```
        Form f = new Form("custom tabs", new BorderLayout());
        Tabs outerTabs = new Tabs();

        Tabs tabs = new Tabs();
        
        tabs.addTab("inner1", new Label("content1"));
        tabs.addTab("inner2", new Label("content2"));
        tabs.addTab("inner3", new Label("content3"));
        
        Container tabsCont = tabs.getTabsContainer();

        TableLayout newLayout = new TableLayout(1, 2);
        tabsCont.getComponentAt(0).setHidden(true);
        tabsCont.setLayout(newLayout);
        
        newLayout.addLayoutComponent(newLayout.cc(0,0).widthPercentage(50), tabsCont.getComponentAt(1), tabsCont);
        newLayout.addLayoutComponent(newLayout.cc(0,1).widthPercentage(50), tabsCont.getComponentAt(2), tabsCont);
        
        outerTabs.addTab("outer1", new Container());
        outerTabs.addTab("outer2", tabs);
        
        
        f.addComponent(BorderLayout.CENTER, outerTabs);
        f.show();
```

I would expect that the the first button among the tab buttons of the inner `Tabs` component is hidden and the other two buttons share the space of their parent equally. But on iOS the buttons are left-aligned and ignore the layout settings completely.

it looks like this (except for iOS):

![Bildschirmfoto 2020-02-11 um 16 14 19](https://user-images.githubusercontent.com/5825645/74249531-98086780-4ce9-11ea-9ee8-ef30521fc8c8.png)

**Comment by codenameone** [comment]
@shannah This looks like a safe region regression

**Comment by shannah** [comment]
Fixed with #3023
